Library/Homebrew/extend/git_repository.rb

```diff
@@ -40,11 +40,12 @@ def git_head
   end
 
   # Gets a short commit hash of the HEAD commit.
-  sig { returns(T.nilable(String)) }
-  def git_short_head
+  sig { params(length: T.nilable(Integer)).returns(T.nilable(String)) }
+  def git_short_head(length: 4)
```

Comment:
Do we have any callers that need `length` or are you just adding it for future use? If the prior: cool, if the latter: might be worth omitting until it's needed.

Reply:
Yes, `Tap#git_short_head` expects `Pathname#git_short_head` to return a string of length 4

It probably would make more sense to make the `length` parameter default to `nil` and have `Tap#git_short_head` pass in a value of `4`